templates/intro-services/main.py

```diff
@@ -0,0 +1,16 @@
+import requests
```

Comment:
Add this to make it a bit more friendly out of the box

Reply:
It was already there, I just created the file so the user doesn't need to copy paste code from the markdown cell.